HW0/P5/P5.txt

```diff
@@ -1 +1,48 @@
 # Your answers here.
+(let's say putting the first number in the calculator takes a second ie 0 + first number)
+
+1. 256 bags / 1 cashier = 256 calculations = 256 seconds
+
+2. 256 bags / 8 cashiers = 32 calculations / cashier = 32 seconds
```

Comment:
I got 34 seconds here, based on:
+16s to get 256 bags -> 128
+8s to get 128 bags -> 64
+4s to get 128 bags -> 32
+2s to get 32 bags -> 16
+1s to get 16 bags -> 8
+1s to get 8 bags -> 4
+1s to get 4 bags -> 2
+1s to get 2 bags -> answer
+
+= 34 seconds